build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'java'
 apply plugin: 'ivy-publish'
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
Android doesn't support v8 these days? Only the very latest?

Reply:
In my new commit this has been changes to 1.7.